providers/class-two-factor-email.php

```diff
@@ -269,7 +269,7 @@ public function authentication_page( $user ) {
 		<p><?php esc_html_e( 'A verification code has been sent to the email address associated with your account.', 'two-factor' ); ?></p>
 		<p>
 			<label for="authcode"><?php esc_html_e( 'Verification Code:', 'two-factor' ); ?></label>
-			<input type="tel" name="two-factor-email-code" id="authcode" class="input" value="" size="20" pattern="[0-9]*" />
```

Comment:
Allow all kinds of inputs to avoid blocking users as they copy codes with wrapping empty spaces or line breaks from their email clients.